### DISCUSSION TITLE
PowMR Inverter (RS485)

### DISCUSSION THREAD

**Discussion by stonejatz** [ai-republic/bms-to-inverter#74] [Q&A]
Thanks for putting together this great project. Like many other Aussie's I've picked up some Samsung batteries which I've been able to read data from using CAN.

I was wondering if my PowerMR POW-VM5K-III inverter would work on RS485? I received this from PowMR 
[powmr-inverter-rs485protocol.pdf](https://github.com/user-attachments/files/17550450/powmr-inverter-rs485protocol.pdf)

**Comment by ai-republic** [comment]
Hi @stonejatz,
thanks for liking this project! I can try and implement the protocol this weekend for you

**Comment by stonejatz** [comment]
Thank you so much, I would appreciate that very much.  If there is anything you need let me know.
